lib/entities/quote/DutchQuote.ts

```diff
@@ -332,6 +385,22 @@ export class DutchQuote implements IQuote {
     }
   }
 
+  public get portionAmountOutStart(): BigNumber {
+    return this.amountOutStart.mul(this.portionBips ?? 0).div(BPS);
+  }
+
+  public get portionAmountOutEnd(): BigNumber {
+    return this.amountOutEnd.mul(this.portionBips ?? 0).div(BPS);
+  }
+
+  public get amountInGasAndPortionAdjusted(): BigNumber {
+    return this.amountIn.add(this.portionAmountOutStart);
```

Comment:
this seems wrong? we shouldnt be adding amountIn with `portionAmountOutStart` right?  As a matter of fact, `this.amountIn` already includes the gas and the fee for exactOut, so thats what you need.

Again, for exactIN, i dont think this metric makes any sense right?